data/overlayfiles/motd-manager-server-byos/etc/motd.finished

```diff
@@ -0,0 +1,30 @@
+SUSE Manager {SUMA_VERSION} Server
+{OS_PRETTY_NAME} {ARCH} (64-bit)
+
+    ____
+  /@    ~-.
+  \/ __ .- |
+   // // @
+
+If the instance is not already registered, please register this instance
+using your existing SUSE entitlement.
+
+As "root" (sudo or sudo -i) use either one of the following commands:
+ - SUSEConnect --url=https://scc.suse.com  -e company@example.com -r YOUR_CODE
+ - yast scc
+
+to register the instance with SCC
+
+ - registercloudguest -r YOUR_CODE
+
+to register the instance against the SUSE update infrastructure.
+
+Without registration this instance does not have access to updates and
+security fixes.
```

Comment:
The registration info is provided by `data/overlayfiles/motd-addon-byos`, which is used for assembling the motd file for the BYOS version. No need to include it here.